File: Python_Ruby/container.rb
# 컨테이너 => 배열

names = ['ujoo', 'dongho', 'jun']
puts(names.class) # Array
puts(names[0]) # ujoo
names[0] = 'u0408'
print(names) # ["u0408", "dongho", "jun"]

#배열 심화

al = ['A', 'B', 'C', 'D']
puts(al.length) # 4
al.push('E')
print(al) # ["A", "B", "C", "D", "E"]
al.delete_at(0)
print("\n")
print(al) # ["B", "C", "D", "E"]

File: Python_Ruby/container.py
# 컨테이너 => 배열

print(type('ujoo')) #<class 'str'>
name = 'ujoo'
print(name) #ujoo
print(type(['ujoo', 'dongho', 'jun'])) #<class 'list'>
names = ['ujoo', 'dongho', 'jun']
print(names) 
print(names[2]) #jun
ujoo = ['programmer', 'seoul', 25, False]
ujoo[1] = 'busan'
print(ujoo) #['programmer', 'busan', 25, False]


#리스트 심화

al = ['A', 'B', 'C', 'D']
print(len(al)) # 4
al.append('E')
print(al) #['A', 'B', 'C', 'D', 'E']
del(al[0])
print(al) #['B', 'C', 'D', 'E']